blocks/author-bio-block/features/author-bio/default.jsx

```diff
@@ -218,52 +179,49 @@ export const AuthorBioItems = ({ arcSite, content }) => {
 					: null;
 
 			// Make the name a hyperlink if a url to the bio page is provided
-			const authorName = original.byline ? (
-				<PrimaryFont as="h2" className="authorName" fontColor="primary-color">
+			const authorName = original?.byline ? (
+				<Paragraph className={`${BLOCK_CLASS_NAME}__authorName`} data-testid="author-name">
 					{original.byline}
-				</PrimaryFont>
+				</Paragraph>
 			) : undefined;
-			const authorNameWithHyperlink = author.url ? (
-				<a href={author.url}>{authorName}</a>
+			const authorNameWithHyperlink = author?.url ? (
+				<a data-testid="author-name" href={author?.url}>
+					{authorName}
+				</a>
 			) : undefined;
 
 			authorList.push(
-				<section key={author.name ? author.name : ""} className="authors">
-					<section className="author">
-						{renderAuthorInfo(author, arcSite)}
-						<section className="descriptions">
-							{authorNameWithHyperlink || authorName}
-							{/* there will always be a description via conditional on 52 */}
-							<p>{author.description}</p>
-							<section className="socialButtons">{socialLinks}</section>
-						</section>
-					</section>
-				</section>
+				<div className={`${BLOCK_CLASS_NAME}__author`} key={author?.name ? author.name : ""}>
+					{renderAuthorImage(author)}
+					<Stack data-testid="descriptions">
```

Comment:
```suggestion
					<Stack>
```

I still see the test id. Can you remove all of the test ids per my previous feedback.

Reply:
addressed it in [65af1f4](https://github.com/WPMedia/arc-themes-blocks/pull/1474/commits/65af1f4c61e8ac222f9fd1117b69979643598a5a)